content/buy-navcoin/index.es.md

```diff
@@ -94,19 +94,19 @@ newTab="true"
     {{< exchange
         titleText="Change Now"
         imgSrc="/images/buy-navcoin/buy-changenow.png"
-        text="Instant Exchange"
+        text="Intercambio instantaneo"
         linkUrl="https://changenow.io/"
     >}}
     {{< exchange
         titleText="Coindirect"
         imgSrc="/images/buy-navcoin/buy-coindirect.png"
-        text="Direct Purchase"
+        text="Compra Directa''
```

Comment:
```suggestion
        text="Compra Directa"
```
There are two single quotes here (`' '`) here instead of a single double quote (`"`)